pkg/rancher-desktop/backend/lima.ts

```diff
@@ -473,7 +479,11 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
       // This shouldn't be possible (it should only be running if we started it
       // in the same Rancher Desktop instance); but just in case, we still stop
       // the VM anyway.
-      await this.lima('stop', MACHINE_NAME);
+      try {
+        await this.lima('stop', MACHINE_NAME);
+      } catch (ex: any) {
+        console.error('Ignoring error trying to stop lima: ', ex);
```

Comment:
Why is this code path suddenly a problem (it's not anywhere near factory reset)?

Furthermore, this should have a unique message (and not duplicate the other one); that makes it clearer where any logged exception is coming from.

Reply:
See first comment on exception handling.